tv_apps/smart-system/fxa/js/fxam_key_navigation.js

```diff
@@ -0,0 +1,108 @@
+/* global KeyNavigationAdapter */
```

Comment:
I know many of this files don't have it, but let's not repeat the same mistake with new ones. Add the MPL2 license header, please.